team/bundles/org.eclipse.compare/compare/org/eclipse/compare/internal/DocLineComparator.java

```diff
@@ -39,6 +43,7 @@ public class DocLineComparator implements ITokenComparator {
 	private boolean fIgnoreWhiteSpace;
 	private ICompareFilter[] fCompareFilters;
 	private char fContributor;
+	private Optional<IIgnoreWhitespaceContributor> fIgnoreWhitespaceContributor;
```

Comment:
could this be made final?

Reply:
yes, thanks for the hint. Done with 9376205.